src/condor_dagman/condor_submit_dag.cpp

```diff
@@ -114,6 +117,17 @@ int main(int argc, char *argv[])
 	tmpResult = dagmanUtils.setUpOptions( deepOpts, shallowOpts, dagFileAttrLines );
 	if ( tmpResult != 0 ) return tmpResult;
 
+		// Post setUpOptions() will determine custom dagman configuration file
+		// If we have a config file then process it for further DAGMan setup options
+	if (!shallowOpts.strConfigFile.empty()) {
+		if (access(shallowOpts.strConfigFile.c_str(), R_OK) != 0 &&
+			!is_piped_command(shallowOpts.strConfigFile.c_str())) {
```

Comment:
Do we need the `access` call?  Often this is a race condition, and the `is_piped_command` has to handle that error condition in any case, so you might as well just have one call.

Reply:
`is_piped_command()` only checks for a passed filename ending in a '|' and returns True or False with no error messages. Thus, both `access` and `is_piped_command` calls are required here at the moment.